polygeist/test/polygeist-opt/uniformity.mlir

```diff
@@ -176,3 +176,69 @@ func.func @test2(%cond: i1, %val: i64, %arg1: memref<?x!sycl_nd_item_2>)  {
 
   return
 }
+
+// -----
+
+!sycl_array_2 = !sycl.array<[2], (memref<2xi64, 4>)>
+!sycl_id_2 = !sycl.id<[2], (!sycl_array_2)>
+!sycl_range_2 = !sycl.range<[2], (!sycl_array_2)>
+!sycl_accessor_impl_device_2 = !sycl.accessor_impl_device<[2], (!sycl_id_2, !sycl_range_2, !sycl_range_2)>
+!sycl_group_2 = !sycl.group<[2], (!sycl_range_2, !sycl_range_2, !sycl_range_2, !sycl_id_2)>
+!sycl_item_base_2 = !sycl.item_base<[2, true], (!sycl_range_2, !sycl_id_2, !sycl_id_2)>
+!sycl_accessor_2_f32_r_gb = !sycl.accessor<[2, f32, read, global_buffer], (!sycl_accessor_impl_device_2, !llvm.struct<(memref<?xf32, 2>)>)>
+!sycl_item_2 = !sycl.item<[2, true], (!sycl_item_base_2)>
+!sycl_nd_item_2 = !sycl.nd_item<[2], (!sycl_item_2, !sycl_item_2, !sycl_group_2)>
+
+gpu.module @device_func {
+
+// COM: Check the uniformity inter-procedurally.
+func.func private @test3(%cond: i1, %uniform_val: i64, %non_uniform_val : i64)  {
+  %alloca = memref.alloca() : memref<10xi64>
+
+  // COM: The stored value is known inter-procedurally to be uniform.
+  // CHECK: test2_load1, uniformity: uniform
```

Comment:
```suggestion
  // CHECK: test3_load1, uniformity: uniform
```
Also the CHECKs below.

Reply:
done I checked the traces and they were all passing but the CHECKs were not updated.